lib/mysql.js

```diff
@@ -249,6 +240,39 @@ MySQL.prototype.updateOrCreate = MySQL.prototype.save =
       callback(err, data, meta);
     });
   };
+  
+/**
+ * Replace if the model instance exists with the same id or create a new instance
+ *
+ * @param {String} model The model name
+ * @param {Object} data The model instance data
+ * @param {Object} options The options
+ * @param {Function} [callback] The callback function
+ */
+MySQL.prototype.replaceOrCreate = function (model, data, options, callback) {
+  var self = this;
+  var fieldsGeneratorFunc = function(model, data) {
+    return self.buildReplaceFields(model, data);
+  };
+  this.modifyOrCreate(model, data, options, fieldsGeneratorFunc, callback);
+};
+
+/**
+ * Update if the model instance exists with the same id or create a new instance
+ *
+ * @param {String} model The model name
+ * @param {Object} data The model instance data
+ * @param {Object} options The options
+ * @param {Function} [callback] The callback function
+ */
+MySQL.prototype.save = MySQL.prototype.updateOrCreate =
+  function(model, data, options, callback) {
+    var self = this;
+    var fieldsGeneratorFunc = function(model, data) {
```

Comment:
No! `modifyOrCreate` is a shared function where you can pass function (which can be anonymous function) to build fields for either update or replace...